python/servo/command_base.py

```diff
@@ -303,6 +303,7 @@ def resolverelative(category, key):
         self.config["build"].setdefault("incremental", None)
         self.config["build"].setdefault("webgl-backtrace", False)
         self.config["build"].setdefault("dom-backtrace", False)
+        self.config["build"].setdefault('use-crown', False)
```

Comment:
Can we remove this? We are gradually trying to move away from `mach` so adding as little as possible to the configuration file will make it easier to do this.

Reply:
Sure, I don't mind. Users hacking on DOM, could just locally `export RUSTC=crown` or edit the `.cargo/config.toml` to avoid manually passing `--use-crown`.

Reply:
removed